### ISSUE TITLE
Sun gain destroys scene

### ISSUE THREAD

**Issue by kobrad** [LuxCoreRender/BlendLuxCore#495]
Hi, using blender 2.83.1, luxcore beta 1 2.4 - new user.

I'm having problems with sun gain, where if I set it to something higher than 0.00005 it destroys texture gamma in the scene and causes the sunlight to appear weaker. After hunting for a global texture gamma setting I finally figured out that I set the sun gain to 0.5, trying for stronger sunlight. Only after opening a backup scene file I saw that before I messed with it it was set to something like 0.00004.

Not sure what this is about, but this setting didn't change the exposure in the scene - my white materials stayed white. But the global lighting was washed out completely and glossy materials became almost white.

This seems counter intuitive to me, but I don't know if this is normal behavior?

**Comment by Theverat** [comment]
Can you show some images and maybe upload a simple testscene that shows the issue?

**Comment by kobrad** [comment]
I can't seem to reproduce this problem - could it be something with opening a file made with luxrender 2.3 ? I've tried creating the same situation from scratch, and everything works as I would expect it to (except the default sun gain being 0.00002).

**Comment by foreachthing** [comment]
I have the same issue with some scence made with previous blender/lux...

**Comment by foreachthing** [comment]
This is Blender default:
(also, when adding a new sun, gain is set to 0.00002)
![image](https://user-images.githubusercontent.com/10420187/86349750-ec44c180-bc61-11ea-9713-cc290ed1edb9.png)

This is with sun gain 1.0:
![image](https://user-images.githubusercontent.com/10420187/86349798-0088be80-bc62-11ea-8ee0-0186264e45fe.png)


Blender: 2.83.0
LuxCore: 2.4b1

**Comment by Theverat** [comment]
The screenshots look like correct behaviour to me?
I mean, this is exactly the reason why the sun gain is set to 0.00002. If we would set it to 1 by default, we would need to set the tonemapper gain to 0.00002 by default, which would make all other lights appear so dark that they would not be visible by default. So we would need to raise their default brightness as well, however we can't do this if Cycles settings are used for some lights, which is important if people want to render existing Cycles scenes with Lux.

**Comment by kobrad** [comment]
That's fine, I didn't know that the default is set that way to be compatible with Cycles. But why doesn't everything go back to normal once we set the sun gain back to almost 0?

**Comment by foreachthing** [comment]
Ok. Here's an old render, rendered in luxrender:
(Gain was set to 1.0 in each and every lightgroup)
![image](https://user-images.githubusercontent.com/10420187/86365766-3afe5580-bc7a-11ea-9982-fedba02bae35.png)
I let it render for 10 minutes.

And this is the old .blend file opened in Blender 2.83 (LuxCore 2.4b1):
![image](https://user-images.githubusercontent.com/10420187/86365904-6da84e00-bc7a-11ea-9030-b143f03c8e35.png)

You can clearly see, that the new image is waaaaaaay too dark.
Textures are missing to... this can be fixed easily! But the sun has not the same impact as it used to.

By the way: WOW on the denoiser! And the overall progress of LuxCore!

This image has **hundreds** of hours behind it:
(rendered on luxrenderer)
![image](https://user-images.githubusercontent.com/10420187/86366568-3a19f380-bc7b-11ea-9ae7-7eccf430ab23.png)

**Comment by Theverat** [comment]
I have the feeling you guys are mixing up two separate issues here.
@foreachthing Please check if your lights and tonemapper have the same brightness settings as in old Lux. If the issue persists, please open a new issue here.

**Comment by Theverat** [comment]
Closing this, if there is still an open question, feel free to re-open.
